_sass/_page.scss

```
@@ -57,11 +57,12 @@ h1 + p {
 }
 
 a {
-  color: #08c;
+  color: #007dbc;
```

Update to match GitHub's link color.

```suggestion
  color: #0366d6;
```